oneclick.py

```diff
@@ -589,6 +682,7 @@ def isbn_info_to_metadata(cls, book, include_bibliographic=True, include_formats
                     subjects.append(subject)
 
             # audience options are: adult, beginning-reader, childrens, young-adult
+            # NOTE: audience can be set to "Adult" while publisher is "HarperTeen".
```

Comment:
Sorry, I don't mean this is set in our systems, I mean this is conflicting data coming in from OneClick.  So, I wanted to reflect this finding, so that if we're surprised by how things end up classified on our end, we'll be one step further in analyzing the cause.  I'll edit the note to clarify.